src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,41 @@
+package command;
+
+import java.io.IOException;
+
+import task.TaskList;
+import task.Task;
+import exception.ScheduloException;
```

Comment:
Perhaps this is in the wrong lexicographical order? 🤔

Reply:
Oh thanks for pointing it out, I will check the ordering